src/components/User/UserProfile/UserProfile.jsx

```diff
@@ -97,7 +97,7 @@ const UserProfile = () => {
 								/>
 							) : (
 								<img
-									src='../../src/assets/icons/empty-avatar.png'
+									src='../../src/assets/icons/account-circle-dark.svg'
```

Comment:
Wydaje mi się, że po przerzuceniu plików do public ścieżka powinna wyglądać tak:
```suggestion
									src='/assets/icons/account-circle-dark.svg'
```